pkg/network/encoding/marshal/format.go

```diff
@@ -300,6 +300,5 @@ func unsafeStringSlice(key string) []byte {
 		return nil
 	}
 	// Reinterpret the string as bytes. This is safe because we don't write into the byte array.
-	sh := (*reflect.StringHeader)(unsafe.Pointer(&key)) //nolint:staticcheck // TODO (WINA) fix reflect.StringHeader has been deprecated: Use unsafe.String or unsafe.StringData instead
-	return unsafe.Slice((*byte)(unsafe.Pointer(sh.Data)), len(key))
+	return unsafe.Slice(unsafe.StringData(key), len(key))
```

Comment:
I'm not 100% sure about this line -- is this changing because unsafe.StringData was a new addition to go ?

Reply:
Yes